pkg/api/validation/validation.go

```diff
@@ -30,12 +30,29 @@ import (
 	"github.com/golang/glog"
 )
 
+const MinPort int = 1
+const MinHostPort int = 0
```

Comment:
ah, of course.  How dumb of me.  Yeah, then I guess all the messages need to be "must be at least %d and less than %d" or "must be between %d and %d, inclusive"